counsel.el

```diff
@@ -1491,6 +1491,9 @@ If NO-ASYNC is non-nil, do it synchronously instead."
                                nil " *counsel-gg-count*")
        nil))))
 
+(defun counsel--normalize-grep-match (str)
+  (if (string-prefix-p "\\.[/\\]" str) str (concat "./" str)))
```

Comment:
Should this be `string-match-p`?

Reply:
hum I don't want to match a path such as "foo./bar"

Reply:
oh got it. string-prefix-p does not handle regexps